### DISCUSSION TITLE
Suggestion for collaboration of data

### DISCUSSION THREAD

**Discussion by swapnilsarwe** [jodal/comics#552] [Ideas]
Hello @jodal ,

@oseparovic maintains a repository of comic selectors at https://github.com/oseparovic/comic_selectors
Although it has a separate purpose that yours, most of the meta information about the comic is very similar.

I was planning to sync both of your repositories in some way. Not asking any of you to change your current structure.
But may be I can write something by which any new things added in either of repositories can be added in the other one conveniently.

Just want to know if you think its worth it & you guys approve it?

**Comment by jodal** [comment]
I still run an instance of this project, but other than that this project is quite inactive. Many of the crawlers are no longer functional and nobody have looked into it and tried to fix any of them for a long time.

This is not a "no, not interested" but more "beware of what you spend your time on".

**Reply by swapnilsarwe** [reply]
Get your point @jodal . Thanks for the advice. Appreciate it.
